tests/components/ring/conftest.py

```diff
@@ -36,6 +38,63 @@ def mock_ring_auth():
         yield mock_ring_auth.return_value
 
 
+@pytest.fixture
+def mock_ring_devices():
+    """Mock Ring devices."""
+
+    devices = get_mock_devices()
+    device_list = list(chain.from_iterable(devices.values()))
+
+    class FakeRingDevices:
+        """Class fakes the RingDevices class."""
+
+        all_devices = device_list
+        video_devices = (
+            devices["stickup_cams"]
+            + devices["doorbots"]
+            + devices["authorized_doorbots"]
+        )
+        stickup_cams = devices["stickup_cams"]
+        other = devices["other"]
+        chimes = devices["chimes"]
+
+        get_device = lambda _, id: [
+            device for device in device_list if device.id == id
+        ][0]
+        get_video_device = lambda _, id: [
+            device
+            for device in device_list
+            if (
+                device.id == id
+                and device.family in {"stickup_cams", "doorbots", "authorized_doorbots"}
+            )
+        ][0]
```

Comment:
Would it make sense to make a function out of this instead of having 4 lambdas?

Reply:
Done